lib/IMAP/MessageMapper.php

```diff
@@ -703,7 +703,6 @@ public function getBodyStructureData(Horde_Imap_Client_Socket $client,
 			$partsQuery = new Horde_Imap_Client_Fetch_Query();
 			if ($htmlBodyId !== null) {
 				$partsQuery->bodyPart($htmlBodyId, [
-					'decode' => true,
```

Comment:
@miaulalala could you document your findings for later reference?

Reply:
Leaving a comment here so I don't forget. Not every IMIP server allows for the decdoing to happen there as it needs capability BINARY